src/App.css

```diff
@@ -3,3 +3,305 @@
+  font-size: 12px;
+}
+
+.LinksActive {
+  color: black !important;
+}
+
+.first-column {
+  display: flex;
+  flex-direction: column;
+  width: 500px;
+}
+
+.crud {
+  display: flex;
+  flex-direction: row;
+}
+
+.School-of {
+  width: 3.063rem;
+  height: 1.125rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.Title {
+  width: 12.313rem;
+  height: 1.813rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.Suzanne-Collins {
+  width: 6.688rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.Comments {
+  width: 4.563rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.Remove {
+  width: 3.375rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background: none !important;
+  border: none;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  cursor: pointer;
+}
+
+.Edit {
+  width: 1.688rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.Line-2 {
+  width: 0.125rem;
+  height: 1.5rem;
+  margin: 0 1.063rem 0 0.938rem;
+  border: solid 1px var(--white-two);
+  color: rgb(241, 241, 241);
+}
+
+.AllAdd {
+  width: 1200px;
+}
+
+.AddBookText {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 22px;
+  color: rgb(125, 125, 125);
+}
+
+.ColumnRow {
+  width: 665px;
+}
+
+.ColumnCheckBox {
+  width: 285px;
+}
+
+.ColumnButton {
+  width: 185px;
+}
+
+.LineVertical {
+  margin-top: 45px;
+  margin-bottom: 35px;
+  background: rgb(220, 220, 220);
+  border-left: 1px solid rgb(220, 220, 220);
+  height: 80px;
+}
+
+.completed {
+  opacity: 0.5;
+  font-size: 0.875rem;
+}
+
+.percent {
+  font-size: 2rem;
+  color: #121212;
+}
+
+.UnderConstruction {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 22px;
+  color: rgb(125, 125, 125);
+}
+
+.circle-wrap {
+  width: 50px;
+  height: 50px;
+  background: #858585;
+  border-radius: 50%;
+  border: 1px solid #858585;
```

Comment:
- [ ] Great job here. It will be nice to slightly increase the pixel value of the circle border so it can look exactly like the design.

The Zeplin Design:
![image](https://user-images.githubusercontent.com/51296741/147373713-ed9e953b-d9e1-489e-8660-a7ebade716df.png)

Your design:
![image](https://user-images.githubusercontent.com/51296741/147373723-d8ec2053-6a87-441e-833c-b165c5a41340.png)

Reply:
Done